sdk/php/src/Dagger.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace DaggerIo;
```

Comment:
In Python, there was already a [dagger](https://pypi.org/project/dagger/) package so I used [dagger-io](https://pypi.org/project/dagger-io/) for the package registry, but after installing it's importable by simply `dagger`. The other package was last updated in 2013 and is unlikely that both will be needed in the same project.

In Node.js there was already a [dagger](https://www.npmjs.com/package/dagger) package (for a server framework) but it supports vendored or namespaced packages so we published it under [@dagger.io/dagger](https://www.npmjs.com/package/@dagger.io/dagger).

In PHP, packages can also have a vendor namespace and there isn't one currently in Packagist so that's not an issue. Even the other package with `dagger` in the name is called [RewriteDagger](https://packagist.org/packages/mazer/rewrite-dagger) which is a mocking library, but even if it were just `Dagger` you can import with a different name (`use ... as ...`).